apps/src/templates/teacherNavigation/SidebarOption.tsx

```diff
@@ -11,30 +11,30 @@ import styles from './teacher-navigation.module.scss';
 
 interface SidebarOptionProps {
   isSelected: boolean;
-  sectionId: number;
-  courseVersionName: string | null;
+  sectionId?: number;
```

Comment:
Can you say more about why the `sectionId` is optional here?

Reply:
Because I changed how the section is loaded. Previously we were waiting for the section to be loaded to show the sidebar. Now we show the sidebar while the section loads.